app/javascript/packs/ChangePassword.js

```diff
@@ -0,0 +1,8 @@
+import {ChangePasswordWrapper} from 'ChangePassword'
```

Comment:
I think it's better import it as `ChangePassword` since the wrapper is just a utility to encapsulate the render function. This way when we stop using the wrapper, we don't have to change anything else, only the import sentence.